pods-framework/pods
# Color picker popover UI

## Description


This PR converts the color picker field into a popover color picker using the `Dropdown` WP react component.

## Related GitHub issue(s)





Fixes #6858 

## Testing instructions



1. Create color field (single and repeatable)
2. See current UI
3. Rebuild DFV with the changes in this PR
4. See new UI :)

## Screenshots / screencast



v2.9.1:
![image](https://user-images.githubusercontent.com/826148/184254548-8ae955d0-7679-41f8-b460-41ec0fb2fdb7.png)

Suggestion:
![image](https://user-images.githubusercontent.com/826148/184261150-924408b1-ac85-49b8-8942-6a2e35818b9f.png)
Newer UI suggesion:
![image](https://user-images.githubusercontent.com/826148/184335842-96e99f47-57e2-46fe-b86c-5d8fbcabb57a.png)

## Changelog text for these changes







## PR checklist
